src/libraries/System.Data.Common/src/System/Data/SQLTypes/SQLGuid.cs

```diff
@@ -117,16 +102,24 @@ public static SqlGuid Parse(string s)
                 return new SqlGuid(s);
         }
 
-
         // Comparison operators
-        private static EComparison Compare(SqlGuid x, SqlGuid y)
+        private static unsafe EComparison Compare(SqlGuid x, SqlGuid y)
```

Comment:
Why does this need to be unsafe now? stackalloc to span does not require it.

Reply:
Because of the `sizeof(Guid)`.